packages/fluttium/lib/src/actions/input_text.dart

```diff
@@ -15,40 +16,57 @@ import 'package:fluttium/fluttium.dart';
 /// ```yaml
 /// - inputText:
 ///     text: "Hello World"
+///     replaceCurrentText: true
 /// ```
 /// {@endtemplate}
 class InputText extends Action {
   /// {@macro input_text}
-  const InputText({
+  InputText({
     required this.text,
+    this.replaceCurrentText = false,
```

Comment:
```suggestion
    this.overwrite = false,
```
nit feel free to ignore.